app/api/modules.py

```diff
@@ -1,11 +1,31 @@
-from flask_rest_jsonapi import ResourceDetail
+from flask_rest_jsonapi import ResourceDetail, ResourceList
 
 from app.api.bootstrap import api
 from app.api.schema.modules import ModuleSchema
 from app.models import db
 from app.models.module import Module
 
 
+class ModuleList(ResourceList):
+    """
+    module list
+    """
+    def before_get(self, *args, **kwargs):
+        """
+        before get method to get the resource id for fetching details
+        :param args:
+        :param kwargs:
+        :return:
+        """
+        kwargs['id'] = 1
+
+    decorators = (api.has_permission('is_super_admin', methods=['POST', 'GET'], id="1"),)
```

Comment:
`id="1"` is static. This should be fixed? @bhaveshAn

Reply:
Actually we do not have to create a new object of Module just 1 object of it is required in App. But have to update it. Thats why it is set to 1 only.

Reply:
OK